src/client/components/pages/entities/related-collections.js

```diff
@@ -29,15 +29,21 @@ function EntityRelatedCollections({collections}) {
 		<Row>
 			<Col>
 				<h2>Related Collections</h2>
-				{collections &&
-				<ul className="list-unstyled">
-					{collections.map((collection) => (
-						<li key={collection.id}>
-							<a href={`/collection/${collection.id}`}>{collection.name}</a> by {' '}
-							<a href={`/editor/${collection.ownerId}`}>{collection.owner.name}</a>
-						</li>
-					))}
-				</ul>
+				{collections && collections.length > 0 ? (
```

Comment:
same as the other file:
```suggestion
				{collections?.length > 0 ? (
```